.github/workflows/ci.yml

```diff
@@ -13,7 +13,13 @@ jobs:
     name: Lint
     runs-on: ubuntu-latest
     steps:
+      - name: Checkout repository
```

Comment:
I think this introduced a Yaml parsing error because of the dash on the next line.